### DISCUSSION TITLE
Password reset, no TC2030 lead

### DISCUSSION THREAD

**Discussion by reedy** [revk/xESP32-Faikout#545] [Q&A]
So I bought a "used"/returned Faikin on Amazon... It is the newer style, and it has a password set (if you returned one to amazon recently in a hard plastic box.. hostname set to `faikin-living-room` let me know)...

I've seen https://github.com/revk/ESP32-Faikin/discussions/273, I have a Tasmotizer and have also seen https://github.com/revk/ESP32-Faikin/blob/cfca7d1ee7cc74d03bef91cac39d70021bf7439b/Manuals/Setup.md

It seems I need a TC2030 lead..

Which from searching.. Is a £30+ part (or maybe £15 from aliexpress/similar.. and however long it takes to arrive) Or basically creating something to hold the pins from something else (dupont?), or trying to solder cables to the pins (as on https://github.com/revk/ESP32-Faikin/discussions/374 etc).

It seems it might be easier to just return it to Amazon at this point, but I'm obviously concious of Amazon just trying to sell it onto someone else.

**Comment by revk** [comment]
OK, does it have two round pads in the back?

My plan, not coded yet, but if we can update it we can add this, is for those to be a factory reset.

So I need to code that feature and then we get it updated.

But that assumes you have it on the WiFi Ok. Let me know.

**Reply by reedy** [reply]
Two round pads in the a&a logo? Yeah, it has those.

The SSID appears pretty quickly, but my phone doesn't seem to want to connect for some reason. But it seemed ok connected to my Mac mini

**Comment by revk** [comment]
OK, if you are able to upgrade to latest beta release, and if you have the two round pads (marked `21`), then upgrade, and then short those pads 3 times without a 3 seconds gap, and it should factory reset.

**Reply by reedy** [reply]
I think this is where the problems begin...

It's not connected to my WiFi (so I'm guessing I can't easily update it; feels like what the WiFi pineapple could be used for!), as the password a isn't one I set, so I can't get past the first screen to do anything obvious with it

**Reply by revk** [reply]
I thought even with password, the WiFi settings should work.

Only issue I know is older code did not like long passwords - if you have a long password, change to short one temporarily to get on line and upgraded.

**Reply by revk** [reply]
Failing that let me know the Amazon order number and date.

**Reply by reedy** [reply]
![Screenshot 2024-11-12 at 12 36 53](https://github.com/user-attachments/assets/20a765a5-082f-4254-827e-0545b51f806e)

I am starting to wonder if it was returned for being dodgy, but Amazon decided to resell it (cause they aren't going to even attempt to test it)...

Doesn't reliably want to connect. I have it on a 12V PSU, with 12V and ground hooked up. Purple light is flashing which I guess is expected as it's not connected to a daikin unit.

`204-9591980-6955530` dated `10 November 2024`

**Reply by revk** [reply]
Let me know if Amazon have cocked that refund up, and order a new one.

**Reply by reedy** [reply]
I'm not sure what Amazon have or haven't done. I've not had any notification from them (yet)...

![Screenshot 2024-11-12 at 15 04 11](https://github.com/user-attachments/assets/9c676bc1-7a2c-412b-bce9-4dd6ca7ed567)

**Reply by revk** [reply]
I think that means fun refund (gift card? is that a refund)? I am not sure they understand VAT.

**Reply by reedy** [reply]
I think Amazon was just being slow.

It wasn't appearing in my gift card balance, but it is now (plus an email at 1511), at the full £27. Thanks.

Do you want me to try and return it to A&A towers at some point? Give you a chance to "refurb" it and maybe find a home for it etc

**Reply by revk** [reply]
Not worth the effort, thanks, I hope replacement works.

**Reply by reedy** [reply]
I've got 4 of the older model ones working fine. I'm planning putting another internal unit in another room in the near future, so was planning ahead for that.

Thanks!
